tidy3d/plugins/expressions/metrics.py

```diff
@@ -40,35 +40,35 @@ class ModeAmp(Metric):
 
     Examples
     --------
-    >>> monitor = ModeMonitor(name="monitor1", freqs=[1.0])
+    >>> monitor = ModeMonitor(name="monitor1", f=[1.0])
     >>> mode_coeff = ModeAmp.from_mode_monitor(monitor)
     >>> data = SimulationData()  # Assume this is a valid SimulationData object
     >>> result = mode_coeff.evaluate(data)
     """
 
     monitor_name: str
-    freqs: FreqArray
+    f: FreqArray
```

Comment:
can f take a float? I feel this would be potentially better for usability (similar to xarray coords). could add it later I suppose, but I feel making users do `f=[f]` when we only support a single frequency feels like it could be improved.

Reply:
Also, could we make it optional? if `None` we just isel the single frequency?

Reply:
pretty sure float works fine, pydantic will coerce into listening on validation

Reply:
if we make it optional we can’t validate for single freq here. do you want that validation to happen somewhere else? in design would be possible along with the other post init stuff. but to me doesnt seem like the right place

Reply:
```python
m = ModeAmp(f=1, monitor_name="")
print(m.f). # prints [1.]
```

Reply:
we can just raise an error when the metric is evaluated? and then the post init validation check should catch it automatically?

Reply:
no strong opinion there other than the fact that the validator would once again not show what actually failed in the log 😆 (except if you look at the stack trace)
but either way it's a bit of a crutch until we have multifrequency figured out for this, so it doesn't really matter